### PR TITLE
Add sfrxETH

### PR DESCRIPTION
**Pull request by 0xJonHoldsCrypto**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/metronome-synth exports TVL: 
        
 
 
    ethereum                  2.20 M
    
    total                    2.20 M

File: projects/metronome-synth/index.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ module.exports = {
         "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
         "0xa8b607Aa09B6A2E306F93e74c282Fb13f6A80452",
         "0xc14900dFB1Aa54e7674e1eCf9ce02b3b35157ba5",
-        "0xac3E018457B222d93114458476f3E3416Abbe38F"
+        "0xac3E018457B222d93114458476f3E3416Abbe38F",
+        "0x691Af94cC63B99bd36173eD6Fb1eF5508b2774ec"
```
Comment (g1nt0ki) [inline projects/metronome-synth/index.js:17]: shouldnt this be `0x4Dbe3f01aBe271D3E65432c74851625a8c30Aa7B`

Comment (0xJonHoldsCrypto) [reply projects/metronome-synth/index.js:17]: apologies - posted the MET asset pool instead of the vaToken - you're correct.

**Comment by llamatester** [comment]
The adapter at projects/metronome-synth exports TVL: 
        
 
 
    ethereum                  2.43 M
    
    total                    2.43 M

**Pull request merged**